assets/app/views/editor/edit-main.js

```diff
@@ -60,6 +63,27 @@ var EditView = Backbone.View.extend({
     this.pageSwitcher.set(childView);
 
     return this;
+  },
+  newPage: function(e) {
+    var self = this,
+        path = prompt('Please the new file path', 'pages/thing.md'),
+        opts = {
+          path: path,
+          content: '---\nyo: dude\nlayout: page\n---\n'
```

Comment:
Have you implemented the defaults.yml file you mentioned in standup, or is that coming later?